test/spec/geo/shared-tests-for-carto-layer-group.js

```diff
@@ -0,0 +1,255 @@
+    container = data.container;
+
+    cartoDbLayer0 = new CartoDBLayer({}, { engine: engineMock });
+    cartoDbLayer1 = new CartoDBLayer({}, { engine: engineMock });
+
+    layersCollection = new LayersCollection([cartoDbLayer0, cartoDbLayer1]);
+
+    layerGroupModelMock = new CartoDBLayerGroup({
+      urls: {
+        'subdomains': [0, 1, 2, 3],
+        'tiles': 'http://{s}.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/{layerIndexes}/{z}/{x}/{y}.png',
+        'grids': [
+          [
+            'http://0.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/0/{z}/{x}/{y}.grid.json',
+            'http://1.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/0/{z}/{x}/{y}.grid.json',
+            'http://2.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/0/{z}/{x}/{y}.grid.json',
+            'http://3.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/0/{z}/{x}/{y}.grid.json'
+          ],
+          [
+            'http://0.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/1/{z}/{x}/{y}.grid.json',
+            'http://1.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/1/{z}/{x}/{y}.grid.json',
+            'http://2.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/1/{z}/{x}/{y}.grid.json',
+            'http://3.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/1/{z}/{x}/{y}.grid.json'
+          ]
+        ],
+        'attributes': [
+          'http://ashbu.cartocdn.com/documentation/api/v1/map/0123456789/0/attributes',
+          'http://ashbu.cartocdn.com/documentation/api/v1/map/0123456789/1/attributes'
+        ]
+      },
+      indexOfLayersInWindshaft: [1, 2]
+    }, {
+      layersCollection: layersCollection
+    });
+  });
+
+  afterEach(function () {
+    document.body.removeChild(container);
+  });
+
+  it('should set the right tile url', function () {
+    layerGroupView = new CartoLayerGroupViewClass(layerGroupModelMock, {
+      nativeMap: nativeMap,
+      mapModel: mapModelMock
+    });
+
+    var actual = getTileUrl(layerGroupView);
+    var expected = 'http://0.ashbu.cartocdn.com/documentation/api/v1/map/0123456789/1,2/{z}/{x}/{y}.png';
+
+    expect(actual).toEqual(expected);
+  });
+
+  describe('interactivity', function () {
+    it('should not enable interaction when there are no interactive layers ', function () {
+      layerGroupView = new CartoLayerGroupViewClass(layerGroupModelMock, {
+        nativeMap: nativeMap,
+        mapModel: mapModelMock
+      });
+
+      expect(layerGroupView.interaction.length).toEqual(0);
+    });
+
+    it('should enable interaction when there are no interactive layers but map model has enabled interaction ', function () {
+      mapModelMock.isFeatureInteractivityEnabled.and.returnValue(true);
+
+      layerGroupView = new CartoLayerGroupViewClass(layerGroupModelMock, {
+        nativeMap: nativeMap,
+        mapModel: mapModelMock
+      });
+
+      expect(layerGroupView.interaction.length).toEqual(2);
```

Comment:
Not very clear why `2` by looking at the test. Perhaps we can get that number from `layerGroupModelMock` / `layersCollection`?